### ISSUE TITLE
MEGAsync Freeze at Login

### ISSUE THREAD

**Issue by Mafiu0** [meganz/MEGAsync#299]
MEGAsync freezes shortly after giving the prompt 'Fetching file list'.

I've tried reinstalling the App as well as deleting the Mega folder in the Application Support, however, it hasn't fixed the problem.

(I'm on Mac OS Mojave 10.14.4)

**Comment by gustavklopp** [comment]
Have you tried to use another account on MEGA?

**Comment by jayloofah** [comment]
I also experience this issue almost at every login. In might case, there may be some interaction between my VPN that is starting up at the same time and might be disrupting MEGA's initial connection.

**Comment by mattw-mega** [comment]
Do you have a sync (or syncs) set up that include large numbers of files?  We have some improvements in the works to make that aspect better.

**Comment by jayloofah** [comment]
Yes, there's about 1.5k files. However, this issue might be more closely tied to the OSX client - I'll report back if this is still an issue on Windows.
